what-i'm-listening-to.txt

```diff
@@ -0,0 +1 @@
+I'm listening to Big Questions by Bill Gates & Rashida Jones!! It's very good.
```

Comment:
Where? Is it a online podcast?

Reply:
I think you can find it online or any podcast player 👍